client/src/components/admin/teams/TeamsList.jsx

```diff
@@ -67,14 +86,22 @@ class TeamsList extends Component {
                   <Card.Content extra>
                     <div>
                       {/* #TODO: implement edit team feature */}
-                      <Button basic color="blue" disabled value={team.teamId}>
+                      <Button
+                        basic
+                        color="blue"
+                        disabled
```

Comment:
'Edit Team' button is currently disabled since Edit functionality is not fully implemented yet